### PR TITLE
Add components

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
### Creating a calculator using React
In this project I:

- Update the readme file
- Created the directory components
- Added components.js file
- Styled my work
- Fixed all the issues required

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import './Calculator.css';
+
+const Calculator = () => (
```
Comment (MahmoudBakr23) [inline src/components/Calculator.js]: - [x] As I highlighted in the review summary 👏 Please take a while of your precious time to convert the components to class-based components at this stage of the project 😄 🙏

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,9 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
```
Comment (MahmoudBakr23) [inline src/App.js]: - [x] Please convert the App component into a class-based component. 🙏 
_Kindly refer to the above comment for more details._

File: README.md [hunk 3]
```diff
@@ -1,71 +1,46 @@
-# Getting Started with Create React App
+# Math Magicians
+In this project I created a simple calculator using React framework.
```
Comment (MahmoudBakr23) [inline README.md:2]: - [x] **OPTIONAL**
It'd be awesome if you attached an image of your app to the README file, it makes it look better and more professional 😄 🙏

File: src/App.css [hunk 4]
```diff
@@ -1,39 +1,43 @@
-.App {
-  text-align: center;
+.Calculator {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  grid-template-rows: repeat(6, 1fr);
+  width: 500px;
+  gap: 1px;
+  padding: 1rem;
```
Comment (MahmoudBakr23) [inline src/App.css:7]: - [x] Please adjust the grid system you built for the calculator layout to be more consistent 😄 Yours looks great, but a few adjustments to follow the given design would make it even better 😉 💪 Kindly adjust the 7, 4, and 1 buttons!
**Yours**
Screenshot (1)
**Original**
calculator

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Isabellanamutenda 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive PR headline, summary, and commit messages
- Workflow followed
- Great job using ES6 features and build the components with arrow functions. However, it's very important to practice class-based components at this specific stage of the project.

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Isabellanamutenda) [reply src/components/Calculator.js]: Resolved

File: src/App.js (hunk 2 above)
Comment (Isabellanamutenda) [reply src/App.js]: Resolved

File: src/App.css (hunk 4 above)
Comment (Isabellanamutenda) [reply src/App.css:7]: Resolved

**Comment by Davidosky007** [comment]
## Hi @Isabellanamutenda 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**